build.gradle

```diff
@@ -2,13 +2,37 @@ plugins {
 	id "java"
 	id "eclipse"
 	id "maven-publish"
-	id "de.abas.esdk" version "0.13.2"
+	id "de.abas.esdk" version "1.2.6"
 }
 
 repositories {
-	maven { url "http://$NEXUS_HOST:$NEXUS_PORT/nexus/content/repositories/$NEXUS_NAME-SNAPSHOT" }
-	maven { url "http://$NEXUS_HOST:$NEXUS_PORT/nexus/content/repositories/$NEXUS_NAME" }
-	maven { url "https://registry.abas.sh/artifactory/abas.maven-public/" }
+    exclusiveContent {
+        forRepository {
+            flatDir {
+                // mapped directory containing homedir libs
+                dirs 'erpsync/homedir/lib'
+            }
+        }
+        filter {
+            // restrict this repository to lookup homedir libs only
+            includeGroup('de.abas.homedir')
+        }
+    }
+    exclusiveContent {
+        forRepository {
+            flatDir {
+                // mapped directory containing clientdir libs
+                dirs 'erpsync/clientdir/lib'
+            }
+        }
+        filter {
+            // restrict this repository to lookup clientdir libs only
+            includeGroup('de.abas.clientdir')
+        }
+    }
+        maven { 
```

Comment:
Please fix indentation.

Reply:
This affects the whole file. Please 4 spaces as base-indent (not 8, not tabs). And align statements being at the same level.